src/services/thread.service.ts

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
[Please merge develop into this branch and create a PR to the develop brach.](https://github.com/SoftwareForScience/jiskefet-api/pull/59#issuecomment-487412712)